product_docs/docs/pgd/4/rel_notes/pgd_4.3.2_rel_notes.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: "Release notes for EDB Postgres Distributed version 4.3.2"
+navTitle: "Version 4.3.2"
+---
+
+EDB Postgres Distributed version 4.3.2 is a minor release of EDB Postgres Distributed 4, which includes bug fixes for issues identified in previous versions.
+
+!!! Note
+    This version is required for EDB Postgres Advanced Server versions 12.15, 13.11, 14.8 and later.
+
+| Component | Version |  Type           |                                                   Description                                                           |
+| --------- | ------- | --------------- | ------------------------------------------------------------------------------------------------------------------------|
+| BDR       | 4.3.2   | Feature         | Fix a bug in handling of memory context in presence of triggers |
+| BDR       | 4.3.2   | Feature         | Add some simplistic memory tracking util |
+| BDR       | 4.3.2   | Feature         | Add a test case for bulkload with triggers on downstream |
```

Comment:
This is not a feature and maybe not even worth mentioning in the release notes. I don't think users get access to our regression tests